src/ophys_etl/decrosstalk/active_traces.py

```diff
@@ -6,18 +7,31 @@
            'mode_robust']
 
 
-def mode_robust(input_data, axis=None, d_type=None):
+def mode_robust(input_data: np.ndarray,
+                axis: Union[None, int] = None) -> Union[np.ndarray,
```

Comment:
FYI `axis` can also be typed as `Optional[int]`. I sort of prefer it because it's clearer that it's nullable.

Reply:
Done